components/script/dom/servoparser/mod.rs

```diff
@@ -585,6 +585,9 @@ impl FetchResponseListener for ParserContext {
                 }
             },
             Some(ContentType(Mime(TopLevel::Text, SubLevel::Xml, _))) => {}, // Handle text/xml
+
+            Some(ContentType(Mime(TopLevel::Application, SubLevel::Xml, _))) => {}, // Handle application/xml
```

Comment:
I don't think moving there is appropriate. That branch is specifically for MIME types that we don't know how to parse. What we can do instead is move the matching of `application/xhtml+xml` up to the branch with `text/xml` and `application/xml`. You can use `Sublevel::Ext("xhtml+xml".to_owned())` to match against a sublevel extension type.